{{cookiecutter.project_slug}}/CONTRIBUTING.md

```diff
@@ -0,0 +1,60 @@
+# Contributing
+
+Contributions are welcome, and they are greatly appreciated! Every little bit
+helps, and credit will always be given.
+
+## Get Started!
+Ready to contribute? Here's how to set up `{{ cookiecutter.project_slug }}` for local development.
+
+* Fork the `{{ cookiecutter.project_slug }}` repo on GitHub.
+* Clone your fork locally:
+
+```
+$ git clone git@github.com:{your_name_here}/{{ cookiecutter.project_slug }}.git
```

Comment:
just to be careful you might want to change this to 
$ git clone --recurse-submodules git@github.com:...

So that if there are submodules they get checked out too. (if there aren't it won't do any harm)

Reply:
Smart